templates/account.html

```diff
@@ -8,24 +8,23 @@
   <section class="p-strip is-shallow">
     <div class="row">
       <div class="col-2">
-        {% if user.image %}
+        {% if account.image %}
           <img class="p-media-object__image--large" src="{{ user.image }}" alt="{{ user.nickname }} account" />
```

Comment:
user.image -> account.image

You'll also need to pass in nickname from the context...